static/css/less/colors.less

```diff
@@ -72,9 +72,9 @@
 @grey-555: hsl(0, 0%, 33%);
 @dark-grey-two: hsl(0, 0%, 35%);
 @grey: hsl(0, 0%, 40%);
-@light-grey: hsl(0, 0%, 60%);
+@mid-grey: hsl(0, 0%, 60%);
```

Comment:
When I search, the autocomplete is possibly too light now due to this change.
![screen shot 2018-12-27 at 1 18 48 pm](https://user-images.githubusercontent.com/148752/50494738-f5ecc980-09d9-11e9-979c-ab83aef2fc75.png)

ReaderStats is also a little unreadable:
![screen shot 2018-12-27 at 1 19 14 pm](https://user-images.githubusercontent.com/148752/50494758-08ff9980-09da-11e9-9971-82ef785cf379.png)

I imagine there are issues in other places too (I'm seeing usages in Tools and data tables....

Please consider reverting this change and renaming these colors rather than switching the variables.

e.g. all instances of light-grey would change to mid-grey and vice versa.

Reply:
Ack, good point -- I'll update these